phoenix-hive-base/pom.xml

```diff
@@ -589,6 +589,7 @@
       <dependency>      
         <groupId>org.apache.calcite</groupId>
         <artifactId>calcite-core</artifactId>
+        <version>1.10.0</version>
```

Comment:
Why was this necessary for this patch ?

Reply:
I ran  `mvn clean compile test-compile site -Dspotbugs.site` without this version and I got the following exception:
`Execution default-site of goal org.apache.maven.plugins:maven-site-plugin:3.7.1:site failed: For artifact {org.apache.calcite:calcite-core:null:jar}: The version cannot be empty.`
 So I ran `mvn dependency:tree` to check the version in phoenix4-hive and added it.

Reply:
This depends on the Hive version.
Could you extract this into a property to make it easier to override ?